mantidimaging/gui/windows/recon/view.py

```diff
@@ -159,6 +163,15 @@ def on_row_change(item, _):
         self.reconHelpButton.clicked.connect(lambda: self.open_help_webpage("reconstructions/index"))
         self.corHelpButton.clicked.connect(lambda: self.open_help_webpage("reconstructions/center_of_rotation"))
 
+        self.hists = [self.image_view.recon_hist, self.image_view.sinogram_hist, self.image_view.projection_hist]
+        self.auto_colour_action = QAction("Auto")
+        self.auto_colour_action.triggered.connect(self.on_change_colour_palette)
+
+        for hist in self.hists:
```

Comment:
At the moment this is only applying to a single image, so there no need to add 'Auto' to all the menus.